README.md

```diff
@@ -1,4 +1,43 @@
 rbase
 =====
 
-rbase gem (working with dbf files) for ruby 1.9.3 (iconv removed, some fixes)
\ No newline at end of file
+rbase gem (working with dbf files).
+
+How to use:
+
+##Creating DBF
+```
```

Comment:
add `ruby` after opening backticks everywhere to use markdown syntax highlighting (\```ruby)